kotlin-js-store/yarn.lock

```diff
@@ -0,0 +1,3513 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
should this file be included? seems like it's generated during the build process..

Reply:
yep